pkg/config/config.go

```diff
@@ -241,8 +241,6 @@ type Config struct {
 	// 2. 'zone' is a special key that indicates the DC location of this tidb-server. If it is set, the value for this
 	// key will be the default value of the session variable `txn_scope` for this tidb-server.
 	Labels map[string]string `toml:"labels" json:"labels"`
-	// EnableGlobalIndex enables creating global index.
```

Comment:
This configuration item does not appear in the document, so maybe we could remove it. Just use session variables to control it.

Reply:
Will it then give an error if included in the config file from now on?

Reply:
I add it to `removedConfig`, maybe it is a more correct way.

There will be no errors in the previous method. Now it will have a Log like 
```
config file pkg/config/config.toml.example contained invalid configuration options: enable-global-index; check TiDB manual to make sure this option has not been deprecated and removed from your TiDB version if the option does not appear to be a typo
```

Not sure which one is better.